torchvision/transforms/v2/_geometry.py

```diff
@@ -1011,6 +1094,28 @@ def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
 
 
 class RandomIoUCrop(Transform):
+    """[BETA] Random IoU crop transformation from
+    `"SSD: Single Shot MultiBox Detector" <https://arxiv.org/abs/1512.02325>`_.
+
+    .. betastatus:: RandomIoUCrop transform
+
+    This transformation requires an image or video data and ``datapoints.BoundingBox`` in the input.
```

Comment:
Let's also mention the need for SanitizeBoundingBoxes:

```suggestion
    This transformation requires an image or video data and ``datapoints.BoundingBox`` in the input.
    
    .. warning::
        In order to properly remove the bounding boxes below the IoU threshold, `RandomIoUCrop` must be followed by `~torchvision.transforms.v2.SanitizeBoundingBoxes`, either immediately after or later in the transforms pipeline.
```